### PR TITLE
ref: write pickle fields as json

### PR DESCRIPTION
**Pull request by asottile-sentry**
this has been dark launched at 10% for quite a while now -- yolo?

### CODE REVIEW

File: src/sentry/db/models/fields/picklefield.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from sentry.db.models.fields import jsonfield
 from sentry.utils import json
 
-PICKLE_WRITE_JSON = False
+PICKLE_WRITE_JSON = True
 VALIDATE_JSON_SAMPLE_RATE = 0.1
```
Comment (mattgauntseo-sentry) [inline src/sentry/db/models/fields/picklefield.py:11]: Would it be worth bumping to to 50 before going full yolo?

Comment (asottile-sentry) [reply src/sentry/db/models/fields/picklefield.py:11]: meh maybe

I'll just put the sample at 100% over the weekend wcgw

**Review by mitsuhiko — APPROVED** [review]
INC-XXX incoming

**Comment by chadwhitacre** [comment]
> INC-XXX incoming

https://getsentry.atlassian.net/browse/INC-400 🐭

**Pull request merged**